app/code/Magento/Customer/Model/AccountManagement.php

```diff
@@ -387,7 +393,8 @@ public function __construct(
         AccountConfirmation $accountConfirmation = null,
         SessionManagerInterface $sessionManager = null,
         SaveHandlerInterface $saveHandler = null,
-        CollectionFactory $visitorCollectionFactory = null
+        CollectionFactory $visitorCollectionFactory = null,
+        AddressRegistry $addressRegistry
```

Comment:
Required parameter can not be placed after optional parameters

Reply:
Thank you, @ishakhsuvarov!
I've added the missung default value for the optional parameter.